### PR TITLE
README update

### PR DESCRIPTION
**Pull request by jwise77**
Removing G+ reference and one last mercurial mention in README

### CODE REVIEW

**Comment by jwise77** [comment]
> Looks good to me. This could also be a good opportunity to put in all of the new contributors and update email addresses in the README, but that's certainly not necessary.

I think that's a good idea to consolidate changes into this PR. I'll search the commit log for people not in the list.

**Comment by jwise77** [comment]
Okay, I went through the commit log and added the missing authors.

**Comment by bwoshea** [comment]
There are three approves, but I propose that we hold off until we're ready to release Version 2.6 - that way we can update the docs and also update the link to the docs (line 19 in the README file).

File: README [hunk 1]
```diff
@@ -45,33 +41,43 @@ list of the people who have recently contributed, in alphabetical order:
...
-   * Shuo Kong		    skong.astro@gmail.com
+   * Shuo Kong              skong.astro@gmail.com
+   * Claire Kopenhafer      clairekope@gmail.com
    * Alexei Kritsuk         akritsuk@ucsd.edu
    * Michael Kuhlen         kuhlen@gmail.com
    * James Larrue           james.larrue@diopolis.com
    * Eve Lee                elee@cita.utoronto.ca
-   * Miao Li		    ml3322@columbia
+   * Miao Li                ml3322@columbia.edu
    * Yuan Li                yuan@astro.columbia.edu
```
Comment (gregbryan) [inline README]: Assuming the FDM module is accepted, can we add it's author:

```suggestion
    * Yuan Li                yuan@astro.columbia.edu
    * Xinyu Li               xinyu.li@columbia.edu
```

Comment (jwise77) [reply README]: Done in a separate commit

**Comment by bwoshea** [comment]
OK, we're good to go here - we're going to send the email out slightly later today!

**Pull request merged**